gatsby-config.js

```diff
@@ -44,7 +44,10 @@ module.exports = {
             }
           },
           {
-            resolve: 'gatsby-remark-responsive-iframe'
+            resolve: 'gatsby-remark-responsive-iframe',
+            options: {
+              plugins: [`gatsby-remark-responsive-iframe`]
+            }
```

Comment:
Looks like the plugin [has a new syntax](https://www.npmjs.com/package/gatsby-remark-responsive-iframe#how-to-use).